src/commands/softsign/import.rs

```diff
@@ -54,7 +57,10 @@ impl Runnable for ImportCommand {
                     .priv_key;
 
                 match private_key {
-                    PrivateKey::Ed25519(pk) => pk.to_seed(),
+                    PrivateKey::Ed25519(pk) => {
+                        // TODO(tarcieri): upgrade Signatory version
+                        ed25519::Seed::from_bytes(pk.to_seed().as_secret_slice()).unwrap()
```

Comment:
Since `tendermint-rs` v0.10 is still linking signatory v0.12, it necessitates these gross thunks.

However, I think it should be pretty easy to update it and cut another release now that the *ring* and Secret Connection code will be out-of-the-way.